src/EntityFramework6.Npgsql/EntityFramework6.Npgsql.nuspec

```diff
@@ -16,7 +16,7 @@
     <language>en-US</language>
     <tags>npgsql postgresql postgres data database entity framework ef orm</tags>
     <dependencies>
-      <dependency id="Npgsql" version="3.1.0" />
+      <dependency id="npgsql" version="3.1.0" />
       <dependency id="EntityFramework" version="6.0.0" />
```

Comment:
You might need to required version 3.1.5 to fix the issue completely since 3.1.0 still uses the old name.